Controllers/Backend/ConnectConfig.php

```diff
@@ -871,6 +928,21 @@ public function getFactory()
     }
 
     /**
+     * @return string
+     */
+    protected function getHost()
```

Comment:
Please move this function in ConfigComponent. After that you can use it here and in ConnectBaseController as well.
It should be also removed from ConnectBaseController.

Modify in all places where it's called.